base-notebook/start.sh

```diff
@@ -27,8 +32,11 @@ if [ $(id -u) == 0 ] ; then
         echo "$NB_USER ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/notebook
     fi
 
+    # Switch to the user's home directory after adjusting any paths
+    cd /home/$NB_USER
```

Comment:
This needs to be aware of what $PWD was before. We don't want the workdir to be set unconditionally to $HOME